.travis.yml

```diff
@@ -15,14 +15,30 @@ before_install:
 # Don't build on tags because we'll already have built the commit.
 jobs:
   include:
-    - name: "Linux build"
+    - name: "Linux build - x86"
       os: linux
+      arch: amd64
       dist: trusty
       env: TARGET="linux"
       if: tag IS blank
       script: scripts/ci.bash
```

Comment:
We can modify `ci.bash` to always build locally now right? We don't need the Docker images anymore.

Reply:
Oh unless we still need the Docker image for an older version of libc? And in that case would the image still work for arm64 since we build it in amd64?

Reply:
On codercom/nbin-centos I get:
```
ldd (GNU libc) 2.17
```

On ubuntu/trusty:
```
ldd (Ubuntu EGLIBC 2.19-0ubuntu6.15) 2.19
```

Although ubuntu/precise might work out:
```
ldd (Ubuntu EGLIBC 2.15-0ubuntu10.18) 2.15
```

Reply:
To be honest we shouldn't need Docker anymore, we can just build via LXD and have the things emulated there if you wanna go local. There's really no point anymore to run Docker.

Reply:
I think that'd be dope.

Reply:
Why do we need LXD, by the way? Would it work to just build directly on the image Travis is providing us (maybe by targeting `precise` so we can drop to 2.15 to match VS Code)?

Reply:
if we still have Precise supported in Travis then I see no harm, we just need to get rid of the docker step.

Reply:
Ahhhh, but we'd still need something to build for Alpine. We could stick to Docker for that. The problem I was hoping to solve by getting rid of Docker is to more easily cache things in Travis (like the Yarn cache) since they're only inside the Docker container unless we mount or copy them out. Maybe I'm thinking about this all wrong?

Reply:
Resolving since we decided to stick with Docker.